plugins/parsers/avro/testdata/union-array/telegraf.conf

```diff
@@ -0,0 +1,25 @@
+[[ inputs.file ]]
+  files = ["./testdata/json-array/message.json"]
```

Comment:
This is the wrong message.json ;)

Reply:
Did you push the fix for this? Ready to hit merge ;)

Reply:
Yeah, but not until I had chased down the 32-64 bit stuff.  Should be good now.